hack/images/integration-tests/Dockerfile

```diff
@@ -12,17 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.17.1
+FROM quay.io/kubermatic/build:go-1.17-node-14-kind-0.11-1
 LABEL maintainer="support@kubermatic.com"
 
-ENV KUBEBUILDER_VERSION=2.3.1 \
```

Comment:
We didn't actually use kubebuilder, we just abused their archive to get kube-apiserver and etcd binaries.